src/domain/event/types.ts

```diff
@@ -1,4 +1,27 @@
+import {
+  CourseFieldsFragment,
+  EventFieldsFragment,
+} from '../../generated/graphql';
+import { ROUTES } from '../app/routes/constants';
+
 export type KeywordOption = {
   id: string;
   name: string;
 };
+
+export type EventFields = EventFieldsFragment | CourseFieldsFragment;
+
+export enum EventType {
+  EVENT = 'event',
+  COURSE = 'course',
+}
+
+export const EVENT_ROUTE_MAPPER = {
+  [EventType.EVENT]: ROUTES.EVENT,
+  [EventType.COURSE]: ROUTES.COURSE,
+};
+
+export const EVENTS_ROUTE_MAPPER = {
```

Comment:
```suggestion
export const EVENTS_ROUTE_MAPPER = EVENT_ROUTE_MAPPER;
```

Reply:
I don't understand this suggestion.

Reply:
They are not quite the same

Reply:
I was just blind! Too many hours reviewing 🤓